lms/static/js/certificates/models/certificate_exception.js

```diff
@@ -24,11 +24,9 @@
                     certificate_generated: '',
                     notes: ''
                 },
-
-               url: function() {
-                    return this.get('url');
+                initialize: function (attributes, options) {
```

Comment:
Can we pass the url in the attributes instead of passing a new options variable?